source/OptimaJet.DWKit.StarterApplication/BackendServicesExtension.cs

```diff
@@ -126,19 +128,13 @@ public static IServiceCollection AddBackgroundServices(this IServiceCollection s
 
         public static IServiceCollection AddAuthenticationServices(this IServiceCollection services, IConfiguration configuration)
         {
-
-
             // JWT Auth disabled for now, because we need to
             // 1. Add an Auth0 Id column to the users table
             // 2. Set the DWKitRuntime.Security.CurrentUser
             // 3. Controller actions are not allowed to have multiple authentication schemes
             // 4. Cookies must be removed.
-            services.AddAuthentication(options =>
-            {
-                options.DefaultScheme = JwtBearerDefaults.AuthenticationScheme;
-                options.DefaultAuthenticateScheme = JwtBearerDefaults.AuthenticationScheme;
-                options.DefaultChallengeScheme = JwtBearerDefaults.AuthenticationScheme;
-            }).AddJwtBearer(options =>
+            services.AddAuthentication()
+            .AddJwtBearer(options =>
```

Comment:
I can still login and pretend to know what I'm doing in the admin area before I realize I'll just break stuff. :)

Reply:
for the dwkit admin, I think we may just want a link from the Scriptoria admin to the URL that the dwkit admin is.
It could be like `dwkit.scriptoria.io/admin` or something like that. 
I don't trust OptimaJet's components to be flexible enough to handle be mounting on different URLs.

Reply:
👍